.env.example

```
@@ -33,8 +33,11 @@ APNS_BUNDLE_ID=
 CDN_BASE_URL=https://assets.domain.tld
 
 # Agent integration (optional)
-AGENT_POOL_URL=
-AGENT_POOL_API_KEY=
+# ASSISTANT_API_URL is the assistant runtime service (convos-assistants).
+# It backs POST /api/v2/agents/join and the /.well-known/agents.json proxy.
+# ASSISTANT_API_KEY is optional — set if the deployed service requires a bearer.
+ASSISTANT_API_URL=
+ASSISTANT_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ASSISTANT_API_KEY key should go before the ASSISTANT_API_URL key

(UnorderedKey)